### PR TITLE
chore(upgrade): bump Golang minor version, add go-bindata and upgrade…

### PR DESCRIPTION
**Pull request by catsoap**
… Go tools

### CODE REVIEW

**Comment by catsoap** [comment]
sorry tezosqa failed it can happen that https://SmartPy.io/smartpy-cli-dev/SmartPy.sh is unavailable :grimacing:

**Comment by catsoap** [comment]
tezosqa image fixed, should be ok now

File: golang/Dockerfile.j2 [hunk 1]
```diff
@@ -75,5 +75,6 @@ RUN echo "Starting...\n" && \
   echo "Successfully installed testfixtures\n" && \
 
   go get golang.org/x/tools/cmd/goimports && \
+  go get github.com/shuLhan/go-bindata/... && \
```
Comment (mremi) [inline golang/Dockerfile.j2:78]: please group packages

Comment (catsoap) [reply golang/Dockerfile.j2]: oupsyes, done !

**Pull request merged**